### ISSUE TITLE
product/packing lookup does nothing when clicking on the single result

### ISSUE THREAD

**Issue by teosarca** [metasfresh/metasfresh-webui-frontend#744]
### Type of issue
Bug

### Steps to reproduce
NOTE: Happens for a product which has only one packing material
* create new sales order
* batch entry: try "Convenience salad..." which has only one packing material, the IFCO6410.
* OK, now click on the field again
* the dropdown is opened with my "Convenience salad..." product.
* click on that single dropdown option
1. nothing happens
2. while checking the Network tab, remark that each time i click on that the frontend is calling typeahead?query=Convenience .... 

### Expected behavior
When clicking on that the dropdown shall close, nothing changed.
That's because after selecting the product usually you are asked to pick the packing materials too (the second lookup), but in this case there was only one option so we are not asking.

Of course, in case the second lookup has more then one option, the second lookup's dropdown shall popup next.

![image](https://cloud.githubusercontent.com/assets/1244701/26029003/94ccb45e-3833-11e7-9d37-15891d3045e2.png)

**Comment by damianprzygodzki** [comment]
Lookups are refactored. We should postpone it and test after new widget will come.

**Comment by metas-mk** [comment]
@damianprzygodzki why postpone? is a functional blocker/ bug.

**Comment by damianprzygodzki** [comment]
@metas-mk Because @Dunkat is at the end of refactoring lookups. We are going to integrate the brand new widgets soon (~1d) and i believe that it will be faster integrated, than fixed and tested with older widgets.

**Comment by teosarca** [comment]
Ok, so it shall be fixed  by https://github.com/metasfresh/metasfresh-webui-frontend/issues/600

**Comment by cadavre** [comment]
@metas-lc tested on new lookups?

**Comment by teosarca** [comment]
To be tested after #600

**Comment by metas-mk** [comment]
Tested. The following issues occurred:
a) First product row, Product chosen, packing item is shown as a drop-down. This product only has 1 Packing Item, the drop-down is shown. Because it's the only line, the user just has to press enter and tab and can record the qty. OK
b) Create the second row with the same product as the first row, Product is chosen, [enter], then the packing dropdown does not appear. NOK (should be same as a) )
c) Create the next row with a product which does not have any Packing material. The Product is chosen. The dropdown for Packing Material is opened but empty. The user cannot get out of that dropdown w/ keyboard. NOK (should be like in case a) press [enter] w/ empty selection.

![image](https://cloud.githubusercontent.com/assets/13365687/26623514/4b52b4e0-45ee-11e7-8f55-65e1a46e66a4.png)

**Comment by Dunkat** [comment]
Fixed:
a,b) when only one item is available, should be selected and no dropdown shown, tested on 'Convenience salad'
c) when product does not have any Packing material press enter to close dropdown

**Comment by metas-rc** [comment]
IT
Tested on webui

* selecting product with only 1 packing material defined 
*  => OK: The dropdown is no longer displayed
* => OK: Press tab and get directly to the qty
* Selecting product with no packing material
* => OK:  I get the message bellow the product that " There is no choice available" 
*=> OK: I press enter, then tab => I get to the qty field
* For product with more than 1 packing material, the default one is created but I can only choose another one using the mous. A followup task is already in progress and will be integrated for the next release.
* For products with packing materials defined, I have no posibility to choose "No packing item"  if I use the batch entry. I can do it, nevertheless, if I use the Add new button.
Talked with Mark and this will also be covered in a follow-up task about harmonizing Add New with Batch entry
